lib/services/centrifuge_service.dart

```diff
@@ -65,10 +124,29 @@ class CentrifugeService with ReactiveServiceMixin {
       messageStreamController.sink.add(userMessage);
     });
 
-    _subscription!.subscribe();
+    subscription.subscribe();
+    if (hasSubscribed(channelSocketId)) {
+      subscription.unsubscribe();
+      return null;
+    }
+    subList[channelSocketId] = subscription;
   }
 
   void dispose() {
     messageStreamController.close();
   }
 }
+// 0:"user_id" -> "6146430f1a5607b13c00bc13"
```

Comment:
Please remove this commented code, the event output is likely to be different events, this is only for channels message event

Reply:
Done